src/types/ShopwareTypes.ts

```diff
@@ -242,9 +242,12 @@ export enum RuleType {
     promotionCustomer = 'promotionCustomerRule',
     promotionCart = 'promotionCartRule',
 }
-export interface RuleAssignmentEntity {
+
+export interface RuleAssignmentEntities {
```

Comment:
Why do you define it as Entities? Usually it is defining a single entity and as soon as you add an [] Array to it the developer knows okay it is more then one.

Reply:
The ruleType cannot be extracted from the Entity created by the TestDataService but has to be decided by the test, so it is not part of the entity object.